unity-renderer/Assets/DCLPlugins/SentryPlugin/SentryPlugin.cs

```diff
@@ -0,0 +1,58 @@
+using DCL;
+using Sentry;
+using UnityEngine;
+public class SentryPlugin : IPlugin
+{
+    public SentryPlugin()
+    {
+        DataStore.i.player.playerGridPosition.OnChange += PlayerGridPositionOnOnChange;
+        DataStore.i.player.otherPlayers.OnAdded += OtherPlayersOnChanged;
+        DataStore.i.player.otherPlayers.OnRemoved += OtherPlayersOnChanged;
+        DataStore.i.player.lastTeleportPosition.OnChange += LastTeleportPositionOnOnChange;
+        DataStore.i.realm.realmName.OnChange += RealmNameOnOnChange;
+    }
+
+    private void RealmNameOnOnChange(string current, string previous)
+    {
+        SentrySdk.ConfigureScope(scope =>
+        {
+            scope.SetTag("Current Realm", current);
+            scope.SetTag("Previous realm", previous);
+        });
+    }
+
+    private void LastTeleportPositionOnOnChange(Vector3 current, Vector3 previous)
+    {
+        SentrySdk.ConfigureScope(scope =>
+        {
+            scope.SetExtra("Current Teleport Position", $"{current.x},{current.y}");
```

Comment:
They say `SetExtra` is deprecated nowadays so we need to use named contexts https://docs.sentry.io/platforms/unity/enriching-events/context/